app/auth/login/index.tsx

```diff
@@ -0,0 +1,111 @@
+import { client } from '@/graphql/client';
+import { LOGIN_MUTATION, ORG_LOGIN_MUTATION } from '@/graphql/mutations/login.mutation';
+import { UserContext } from '@/hook/useAuth';
+import { useMutation } from '@apollo/client';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { useRouter } from 'expo-router';
+import { useContext, useState } from 'react';
+import { ToastAndroid, Alert } from 'react-native';
+
+export default function SignInOrganization() {
+  const router = useRouter();
+  const [orgLoginSuccess, setOrgLoginSuccess] = useState(false);
+  const [orgLoginMutation] = useMutation(ORG_LOGIN_MUTATION);
+  const [loading, setLoading] = useState(false);
+  const { login } = useContext(UserContext) || {};
+  const [LoginUser] = useMutation(LOGIN_MUTATION);
+
+  const onOrgSubmit = async (values: any) => {
+    console.log('Submitting values:', values);
+    try {
+      await orgLoginMutation({
+        variables: {
+          orgInput: {
+            name: values.organization,
+          },
+        },
+        onCompleted(loginOrg) {
+          console.log('Login successful:', loginOrg);
+          setOrgLoginSuccess(true);
+        },
+        onError(err: any) {
+          ToastAndroid.show(`${err}`, ToastAndroid.LONG);
+        },
+      });
+    } catch (err) {
+      console.log('Mutation error:', err);
+    }
+  };
+
+  const onSubmit = async (userInput: any) => {
+    try {
+      const orgToken = await AsyncStorage.getItem('orgToken');
+      userInput.orgToken = orgToken;
+      setLoading(true);
+      await LoginUser({
+        variables: {
+          loginInput: userInput,
+        },
+        onCompleted: async (data) => {
+          if (data.addMemberToCohort) {
+            ToastAndroid.show(`${data.addMemberToCohort}`, ToastAndroid.LONG);
+          }
+          if (login && data.loginUser) {
+            login(data.loginUser);
+            await client.resetStore();
+            Alert.alert('Welcome');
+          }
+
+          if (data.loginUser) {
+            const role = data.loginUser.user.role;
+            switch (role) {
+              case 'superAdmin':
+                router.push('Organizations');
+                break;
+              case 'admin':
+              case 'coordinator':
+                router.push('Trainees');
+                break;
+              case 'manager':
+                router.push('Dashboard');
+                break;
+              case 'ttl':
+                router.push('TTL_Trainees');
+                break;
+              default:
+                router.push('Performance');
+                break;
+            }
+          } else {
+            router.push('Dashboard');
+          }
+        },
+        onError: (err) => {
+          if (err.networkError) {
+            Alert.alert('There was a problem contacting the server');
+            ToastAndroid.show('There was a problem contacting the server', ToastAndroid.LONG);
```

Comment:
This will show an alert and a toast at the same time? You should use one only

Reply:
**Done**